### PR TITLE
Changed to switch/case syntax

### PR DESCRIPTION
**Pull request by doot0**
This syntax is a little easier to read. Also cache one value.

### CODE REVIEW

File: app/src/main/main.js [hunk 1]
```diff
@@ -293,15 +293,22 @@ ipcMain.on('move-cropper-window', (event, data) => {
...
-    position[0] -= data.amount;
-  } else if (data.direction === 'up') {
-    position[1] -= data.amount;
-  } else if (data.direction === 'right') {
-    position[0] += data.amount;
-  } else if (data.direction === 'down') {
-    position[1] += data.amount;
+  const amount = data.amount;
+  
+  switch(data.direction){
```
Comment (CodeTheory) [inline app/src/main/main.js]: Please add a space between the switch and the parentheses and the parentheses and the bracket.

File: app/src/main/main.js [hunk 2]
```diff
@@ -293,15 +293,22 @@ ipcMain.on('move-cropper-window', (event, data) => {
...
+      break;
+    case 'up':
+      position[1] -= amount;
+      break;
+    case 'right':
+      position[0] += amount;
+      break;
+    case 'down':
+      position[1] += amount;
+      break;
```
Comment (CodeTheory) [inline app/src/main/main.js]: Please add a `default` statement to this. It'll never happen but maybe add `console.error('No direction detected');` or something to appease the code linter.

File: app/src/main/main.js [hunk 3]
```diff
@@ -293,15 +293,22 @@ ipcMain.on('move-cropper-window', (event, data) => {
...
   }
-
+
```
Comment (CodeTheory) [inline app/src/main/main.js]: Remove trailing whitespace.

File: app/src/main/main.js [hunk 4]
```diff
@@ -293,15 +293,22 @@ ipcMain.on('move-cropper-window', (event, data) => {
...
-  if (data.direction === 'left') {
-    position[0] -= data.amount;
-  } else if (data.direction === 'up') {
-    position[1] -= data.amount;
-  } else if (data.direction === 'right') {
-    position[0] += data.amount;
-  } else if (data.direction === 'down') {
-    position[1] += data.amount;
+  const amount = data.amount;
+
```
Comment (CodeTheory) [inline app/src/main/main.js]: Remove trailing space.

**Comment by doot0** [comment]
@CodeTheory Appeasing the lint-god now.

File: app/src/main/main.js [hunk 5]
```diff
@@ -294,8 +294,8 @@ ipcMain.on('move-cropper-window', (event, data) => {
 
   const position = cropperWindow.getPosition();
   const amount = data.amount;
-  
-  switch(data.direction){
+
+  switch(data.direction) {
```
Comment (CodeTheory) [inline app/src/main/main.js]: needs a space between `switch` and `(data.direction)` too

Comment (doot0) [inline app/src/main/main.js]: Added

**Comment by doot0** [comment]
cc @CodeTheory @matheuss merge?

**Comment by doot0** [comment]
🔥

**Pull request merged**